src/components/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { NavLink, useLocation } from 'react-router-dom';
+
+const Navbar = () => {
+  const location = useLocation();
+
+  return (
+    <header className="header">
+      {location.pathname !== '/' && (
+      <NavLink to="/">
+        <FontAwesomeIcon className="arrow" icon={faAngleLeft} />
+      </NavLink>
+      )}
+      <h3>Crypto Coins</h3>
```

Comment:
- [ ] As I mentioned in the above comment, you should adhere to the `design` guidelines because as per the `design` guidelines your application `font-family` should use `Gill sans` fonts for the `body` and the `title` fonts. Kudos you did great work by using the `Lato` for the rest of the part of the application.

![image](https://github.com/tawakuliKH/coinCap-React/assets/66476329/452f3f00-ef20-4647-8044-84f276c679d5)